src/PointerEvent.js

```diff
@@ -87,6 +87,35 @@ function PointerEvent(inType, inDict) {
   e.pointerType = inDict.pointerType || '';
   e.hwTimestamp = inDict.hwTimestamp || 0;
   e.isPrimary = inDict.isPrimary || false;
+
+  var rect;
+  function boundingClientRect() {
+    if (rect) {
+      return rect;
+    }
+    return (rect = inDict.target.getBoundingClientRect());
+  }
+
+  // CSSOM View Module's extension to mouse events
+  if (inDict.offsetX === undefined) {
+    Object.defineProperty(e, 'offsetX', {
+      get: function() {
+        return e.pageX - boundingClientRect().left;
```

Comment:
Thanks for the review. Can you help fix the calculation to take the padding edge into account?